### PR TITLE
Pipelines page — icon/graph tweaks

### PR DESCRIPTION
**Pull request by sj26**
Show scheduled builds as features build, and in the graph:

screenshot 2016-10-10 14 35 26

Also tweaks colours and icons slightly.

/cc @toolmantim

### CODE REVIEW

File: app/components/organization/Pipeline/graph.js [hunk 1]
```diff
@@ -191,7 +197,7 @@ export default Relay.createContainer(Graph, {
     pipeline: () => Relay.QL`
       fragment on Pipeline {
         id
-        builds(first: 30, branch: "%default", state: [ BUILD_STATE_RUNNING, BUILD_STATE_PASSED, BUILD_STATE_FAILED, BUILD_STATE_CANCELED, BUILD_STATE_CANCELING, BUILD_STATE_BLOCKED ]) {
+        builds(first: 30, branch: "%default", state: [ BUILD_STATE_SCHEDULED, BUILD_STATE_RUNNING, BUILD_STATE_PASSED, BUILD_STATE_FAILED, BUILD_STATE_CANCELED, BUILD_STATE_CANCELING, BUILD_STATE_BLOCKED ]) {
```
Comment (toolmantim) [inline app/components/organization/Pipeline/graph.js]: Is this now ALL THE STATES?

**Comment by keithpitt** [comment]
Instead of the 2 different grays, what if they were little empty boxes (if there wasn't a build there?)

File: app/components/organization/Pipeline/status.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ export default Relay.createContainer(Status, {
     pipeline: () => Relay.QL`
       fragment on Pipeline {
         id
-        builds(first: 1, branch: "%default", state: [ BUILD_STATE_RUNNING, BUILD_STATE_PASSED, BUILD_STATE_FAILED, BUILD_STATE_CANCELED, BUILD_STATE_BLOCKED ]) {
+        builds(first: 1, branch: "%default", state: [ BUILD_STATE_SCHEDULED, BUILD_STATE_RUNNING, BUILD_STATE_PASSED, BUILD_STATE_FAILED, BUILD_STATE_CANCELED, BUILD_STATE_BLOCKED ]) {
```
Comment (keithpitt) [inline app/components/organization/Pipeline/status.js]: You'll also need to update the Relay preload to include this state.

Comment (sj26) [inline app/components/organization/Pipeline/status.js]: Yeah! I can do that when bumping the commit at the other end?

**Comment by ticky** [comment]
Hell yeah, tests!

**Comment by sj26** [comment]
@keithpitt I don't mind! Up to you and @toolmantim.

**Comment by sj26** [comment]
So if pipeline status = last build running then it looks like:

screenshot 2016-10-10 15 32 09

**Comment by sj26** [comment]
And if we do that then we probably want "either the last finished build, or the last build in any state" for a nicer on-boarding state which I can try to figure out in GraphQL.

**Pull request merged**